src/harness/testcases/WINNF_FT_S_SIQ_testcase.py

```diff
@@ -841,6 +842,232 @@ def test_WINNF_FT_S_SIQ_11(self):
+        ppa_record2, [pal_record_1], pal_low_frequency2, pal_high_frequency2,
+        pal_user_id2) 
+    
+    ppa_cluster_list_1 = [0]
+    ppa_cluster_list_2 = []
+    # Move device_b inside the  PPA zone
+    device_b['installationParam']['latitude'] = 38.82767
+    device_b['installationParam']['longitude'] = -97.20497
+
+    # Creating conditionals for Cat B devices
+    self.assertEqual(device_b['cbsdCategory'], 'B')
+    conditionalParameters = {
+        'cbsdCategory': device_b['cbsdCategory'],
+        'fccId': device_b['fccId'],
+        'cbsdSerialNumber': device_b['cbsdSerialNumber'],
+        'airInterface': device_b['airInterface'],
+        'installationParam': device_b['installationParam'],
+        'measCapability': device_b['measCapability']
+    }
+    del device_b['cbsdCategory']
+    del device_b['airInterface']
+    del device_b['installationParam']
+
+    # N2 spectrum inquiry requests
+    # 1. Spectrum Inquiry: 3550-3700 .
+    spectrum_inquiry_1 = json.load(
+        open(os.path.join('testcases', 'testdata', 'spectrum_inquiry_0.json')))
+    spectrum_inquiry_1['inquiredSpectrum'] = [{
+        'lowFrequency': 3550000000,
+        'highFrequency': 3580000000
+    }]
+
+    # 2. Spectrum Inquiry: Invalid Value of low and high frequency and invalid id for cbsdId.
+    spectrum_inquiry_2 = json.load(
+        open(os.path.join('testcases', 'testdata', 'spectrum_inquiry_0.json')))
+    spectrum_inquiry_2['inquiredSpectrum'] = [{
+        'lowFrequency': 3610000000,
+        'highFrequency': 3590000000
+    }]
+    spectrum_inquiry_2['cbsdId'] = "INVALID_CBSD_ID_12345"
+
+    # 3. Spectrum Inquiry: Missing parameter low frequency and missing cbsdId
+    spectrum_inquiry_3 = json.load(
+        open(os.path.join('testcases', 'testdata', 'spectrum_inquiry_0.json')))
+    spectrum_inquiry_3['inquiredSpectrum'] = [{
+        'highFrequency': 3700000000
+    }]
+    spectrum_inquiry_3['cbsdId'] = "REMOVE"
+
+    # Create the actual config.
+    devices = [device_b, device_c, device_e]
+    conditionals = {
+        'registrationData': [conditionalParameters]
+    }
+
+    config = {
+        'registrationRequestsN2': devices,
+        'conditionalRegistrationData' : conditionals,
+        'expectedResponseCodes': [(0,), (103,), (102,)],  
+        'palRecordsN3':[pal_records1,pal_records2],
+        'ppaRecordsN3':[{'ppaRecord':ppa_record1,
+                         'ppaClusterList': ppa_cluster_list_1},
+                        {'ppaRecord':ppa_record2,
+                         'ppaClusterList': ppa_cluster_list_2}],
+   
+        'spectrumInquiryRequestsN2':[spectrum_inquiry_1,spectrum_inquiry_2,spectrum_inquiry_3],
+        'gwpzRecordsN1':[gwpz_e],
+        'fr1Cbsd': [ [{'frequency' :{'lowFrequency': 3550000000,'highFrequency': 3560000000}}, 
+                       {'frequency' :{'lowFrequency': 3560000000,'highFrequency': 3570000000}}, 
+                       ],[],[] ], 
```

Comment:
i'll have to look at this again tomorrow with fresh eyes, but I think the values for fr1 are incorrect.

From the TC:

o	None of the frequencyRange parameters in the availableChannel parameters marked as available for GAA use overlap with the frequency range(s) specified in FR1cbsd,n2, where FR1cbsd,n2 is a list of the frequency ranges specifed in the configuration file for n2-th CBSD (1≤ n2 ≤ N2), and is consistent with the information of protected entities

The only protected entity near the CBSD is the PPA that we moved it into, so shouldn't fr1 in this case be 3570-3580 (the frequency of the PPA)?

Reply:
We have address this issue in the update submit. Modified FR1cbsd to be consistent with the protected entities.As here, the only protected entity where the CBSD belongs is PPA, made FR1cbsd to be consistent with PPA frequency ranges.